website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,195 @@
+---
```

Comment:
The Consul Enterprise LTS docs have sections that aren't present in the Vault Ent LTS docs:
- [Fix eligibility](https://developer.hashicorp.com/consul/docs/enterprise/long-term-support#fix-eligibility)
- [Version compatibility with dependencies](https://developer.hashicorp.com/consul/docs/enterprise/long-term-support#version-compatibility-with-dependencies)

If Vault has such limitations on its LTS offering, they should be explicitly called out

Reply:
I have not seen anything regarding version compatibility and the only fix eligibility I am aware of is "critical patches" which is already called out.

Reply:
We don't have this level of specificity on Vault